docs/setup-cluster/slurm/slurm-known-issues.rst

```diff
@@ -381,6 +381,10 @@ Some constraints are due to differences in behavior between Docker and Singulari
  PBS Known Issues
 ******************
 
+-  If the ``Cluster`` tab on the DeterminedAI WebUI does not display the GPU information, there
+   could be an issue in the PBS configuration. Please refer to :ref:`PBS Requirements
+   <pbs-config-requirements>` to ensure PBS is configured properly.
```

Comment:
Where in the `PBS Requirements` (https://docs.determined.ai/latest/setup-cluster/slurm/slurm-requirements.html#pbs-requirements) would a user find the step necessary to get the GPU information displayed?

Reply:
PBS Requirements lists the important configuration steps necessary along with commands and sample outputs. That should help the users to ensure PBS is configured properly.

Reply:
Updated the ref link to point to the exact section for solving the issue.